cmake/cmake_celix/ContainerPackaging.cmake

```diff
@@ -408,7 +408,7 @@ endfunction()
 Add the selected bundles to the Celix container. These bundles are (if configured) copied to the container build dir and
 are added to the configuration properties so that they are installed and started when the Celix container executed.
 
-The Celix framework support 7 (0 - 6) run levels. Run levels can be used to control the start and stop order of bundles.
+The Celix framework supports 7 (0 - 6) run levels. Run levels can be used to control the start and stop order of bundles.
```

Comment:
Is it possible to make this arbitrary? So that any number of levels is supported? Iirc in Java it is MAX_INT.

Reply:
Yeah I think that is possible, but because the main function for a celix container is generated  with cmake (generator-expression), I am afraid this will be complex. So for now I would like to keep it at the fixed runtime levels.